### PR TITLE
DEVDOCS-5284 [new]: Shipping Provider API, add digital wallets

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-5284]

## What changed?
* Add paragraph about how Shipping Provider APIs should handle incomplete postal codes (from digital wallets)

@bigcommerce/team-shipping 

[DEVDOCS-5284]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5284?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by benpratt77 — APPROVED** [review]

**Review by agchan12 — DISMISSED** [review]

**Comment by bc-andreadao** [comment]
Thanks @agchan12 @benpratt77

File: docs/api-docs/store-management/shipping/shipping-provider-api.mdx [hunk 1]
```diff
@@ -496,6 +496,8 @@ When you uninstall an app with an associated shipping carrier, you also automati
   The response payload will display shipping quotes from lowest to highest price.
 </Callout>
 
+Shipping provider integrations should plan to handle incomplete postcodes when handling shipping quotes. This occurs when stores use [digital wallets](https://support.bigcommerce.com/s/article/Online-Payment-Methods?language=en_US#digital-wallet) as a payment method. Digital wallets truncate postcodes for shipping addresses in UK and Canada. They only provide the first 3 characters to shipping provider integrations.
```
Comment (bc-traciporter) [inline docs/api-docs/store-management/shipping/shipping-provider-api.mdx:499]:
```suggestion
Shipping provider integrations should plan to handle incomplete postcodes when handling shipping quotes. Incomplete postcodes can occur when stores use [digital wallets](https://support.bigcommerce.com/s/article/Online-Payment-Methods?language=en_US#digital-wallet) as a payment method. Digital wallets truncate postcodes for shipping addresses in the UK and Canada as they only provide the first three characters to shipping provider integrations.
```

**Review by bc-traciporter — COMMENTED** [review]
I left a comment to improve the readability. I am open to other suggestions.

**Review by bc-traciporter — APPROVED** [review]

File: docs/api-docs/store-management/shipping/shipping-provider-api.mdx [hunk 2]
```diff
@@ -496,6 +496,8 @@ When you uninstall an app with an associated shipping carrier, you also automati
   The response payload will display shipping quotes from lowest to highest price.
 </Callout>
 
+Shipping provider integrations should plan to handle incomplete postcodes when handling shipping quotes. Incomplete postcodes can occur when stores use [digital wallets](https://support.bigcommerce.com/s/article/Online-Payment-Methods?language=en_US#digital-wallet) as a payment method. Digital wallets truncate postcodes for shipping addresses in the UK and Canada as they only provide the first three characters to shipping provider integrations.
```
Comment (vborinsk) [inline docs/api-docs/store-management/shipping/shipping-provider-api.mdx:499]: Is it worth putting this somewhere more prominently or make it easier to spot like in an info/warning style panel?

I would also add that postcodes are truncated only at time of shipping quotation, and are subsequently available/complete once the order is placed (knowing that some shipping providers also read the Order data after order placement to pull details/update costs).

Comment (bc-andreadao) [reply docs/api-docs/store-management/shipping/shipping-provider-api.mdx:499]: Thanks @vborinsk for the feedback. I created [PR 2074](https://github.com/bigcommerce/dev-docs/pull/2074) for this if you can approve

**Pull request merged**